config/locales/diaspora/en.yml

```diff
@@ -427,9 +427,9 @@ en:
       restrict_posts_i_see_q: "Can I restrict the posts in my stream to just those from certain aspects?"
```

Comment:
No. That's a valid sentence. Do you think there should be a word after 'those'? You can think of it being used like 'ceux' in French - if you've already defined the term that 'those' points to ('posts in my stream' in this case), you don't need to repeat the term.